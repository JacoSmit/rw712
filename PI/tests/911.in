10
7
15 95 98 39 86 8 10
7
2 10 46 90 72 16 98
7
18 20 91 35 28 2 98
7
99 4 69 6 10 63 94
7
96 92 5 38 6 40 12
7
1 98 96 2 11 37 34
7
22 90 20 94 69 16 5
7
18 81 95 28 95 2 17
7
1 93 97 25 49 19 13
7
86 10 99 96 7 7 51
