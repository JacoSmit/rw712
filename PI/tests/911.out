1:
1 ~ 0
2 ~ 32
3 ~ 32
4 ~ 0
5 ~ 32
6 ~ 0
7 ~ 0
2:
1 ~ 0
2 ~ 4
3 ~ 4
4 ~ 28
5 ~ 28
6 ~ 4
7 ~ 36
3:
1 ~ 6
2 ~ 10
3 ~ 28
4 ~ 20
5 ~ 12
6 ~ 2
7 ~ 36
4:
1 ~ 34
2 ~ 2
3 ~ 22
4 ~ 6
5 ~ 10
6 ~ 10
7 ~ 30
5:
1 ~ 36
2 ~ 28
3 ~ 4
4 ~ 16
5 ~ 4
6 ~ 16
7 ~ 12
6:
1 ~ 0
2 ~ 32
3 ~ 32
4 ~ 0
5 ~ 16
6 ~ 16
7 ~ 16
7:
1 ~ 0
2 ~ 32
3 ~ 0
4 ~ 32
5 ~ 32
6 ~ 0
7 ~ 0
8:
1 ~ 0
2 ~ 32
3 ~ 32
4 ~ 0
5 ~ 32
6 ~ 0
7 ~ 0
9:
1 ~ 0
2 ~ 32
3 ~ 32
4 ~ 8
5 ~ 24
6 ~ 8
7 ~ 8
10:
1 ~ 32
2 ~ 0
3 ~ 32
4 ~ 32
5 ~ 0
6 ~ 0
7 ~ 0
